rt/test/TESTOPR2.MLC

```diff
@@ -0,0 +1,1443 @@
+         MNOTE 'ISHEX(''123456789'')=&ISHEX' 
+         AIF   (&ISHEX NE 0).ISERR
+***********************************************************************
+*        TODO: Next test now produces
+*                MNOTE 8,'ISHEX invalid operand value; length is zero'
+*              on z390, as it should. Need to add test to error tests
+* 
+*&ISHEX   SETA  ISHEX('')                      error condition
+*         MNOTE 'ISHEX('''')=&ISHEX' 
+*         AIF   (&ISHEX NE 0).ISERR
+*----------------------------------------------------------------------
+*        z390 tests
+
+&ISHEX   SETA  ISHEX('F0') 
+         MNOTE 'ISHEX(''F0'')=&ISHEX' 
+         AIF   (&ISHEX NE 1).ISERR
+&ISHEX   SETA  ISHEX('+F0')                    non-hex char 
+         MNOTE 'ISHEX(''+F0'')=&ISHEX' 
+         AIF   (&ISHEX NE 0).ISERR 
+&ISHEX   SETA  ISHEX('FG') 
+         MNOTE 'ISHEX(''FG'')=&ISHEX' 
+         AIF   (&ISHEX NE 0).ISERR
+&ISHEX   SETA  ISHEX('000000000')              too many chars; 8 max
+         MNOTE 'ISHEX(''000000000'')=&ISHEX' 
+         AIF   (&ISHEX NE 0).ISERR
+*----------------------------------------------------------------------
+         MNOTE '********** End   ISHEX tests **********'
+*======================================================================
+         MNOTE '********** Begin ISSYM tests **********'
+*----------------------------------------------------------------------
+*        HLASM LR examples
+
+&ISSYM   SETA  ISSYM('Abcd_1234') 
+         MNOTE 'ISSYM(''Abcd_1234'')=&ISSYM' 
+         AIF   (&ISSYM NE 1).ISERR 
+&ISSYM   SETA  ISSYM('_Abcd1234') 
+         MNOTE 'ISSYM(''_Abcd1234'')=&ISSYM' 
+         AIF   (&ISSYM NE 1).ISERR 
+&ISSYM   SETA  ISSYM('##@$_') 
+         MNOTE 'ISSYM(''##@$_'')=&ISSYM' 
+         AIF   (&ISSYM NE 1).ISERR 
+&ISSYM   SETA  ISSYM('1234_Abcd')              invalid initial char
+         MNOTE 'ISSYM(''1234_Abcd'')=&ISSYM' 
+         AIF   (&ISSYM NE 0).ISERR
+***********************************************************************
+*        TODO: Next test now produces
+*                MNOTE 8,'ISSYM invalid operand value; length is zero'
+*              on z390, as it should. Need to add test to error tests
+* 
+*&ISSYM   SETA  ISSYM('')                      error condition
+*         MNOTE 'ISSYM('''')=&ISSYM' 
+*         AIF   (&ISSYM NE 0).ISERR
+*----------------------------------------------------------------------
+*        z390 tests
+
+&ISSYM   SETA  ISSYM('AZ090#$_') 
+         MNOTE 'ISSYM(''AZ090#$_'')=&ISSYM' 
+         AIF   (&ISSYM NE 1).ISERR 
+&ISSYM   SETA  ISSYM('AZ090#$_**') 
+         MNOTE 'ISSYM(''AZ090#$_**'')=&ISSYM' 
+         AIF   (&ISSYM NE 0).ISERR 
+.SKPIS ANOP
+*----------------------------------------------------------------------
+         MNOTE '********** End   ISSYM tests **********'
+***********************************************************************
+         MBOX  'Test AND, NOT, OR, XOR'
+*AGO .SKPLOG
```

Comment:
is this still needed?

Reply:
I left commented AGOs in so that I could uncomment them when testing TESTOPR2.MLC. Happy to go through and delete them now if you think it is the right thing to do.